recipe/meta.yaml

```diff
@@ -18,18 +18,43 @@ requirements:
     - python
     - pip
   run:
-    - python
+    - entrypoints
+    - ipython
+    - jsonschema
+    - numpy
     - pandas
-    - vega 0.4.4
-    - traitlets >=4.3.1
+    - python 
+    - six
+    - toolz
+    - typing
 
 test:
+  source_files:
+    - altair
   requires:
+    - docutils
+    - jinja2
     - pytest
+    - sphinx
+    - vega_datasets
+  commands:
+    - python -m pytest --pyargs --doctest-modules altair
```

Comment:
Are we running full test suites as part of conda-forge recipes now?

Reply:
Yep. The `noarch` packages build pretty quickly and the tests are not that expensive, so I though it was a good idea to run them in the `rc` branch on order to catch dependencies issues, etc.